### PR TITLE
Support old alerting rule syntax

### PR DESCRIPTION
**Pull request by fabxc**
@matthiasr

### CODE REVIEW

File: promql/parse_test.go [hunk 1]
```diff
@@ -1333,12 +1332,6 @@ var testStatement = []struct {
 	}, {
 		input: `foo{a!~"b"} = bar`,
 		fail:  true,
-	}, {
-		input: `ALERT SomeName IF time() LABELS {} 
-			SUMMARY "Global request rate low"
-			DESCRIPTION "The global request rate is low"
-		`,
-		fail: true,
```
Comment (matthiasr) [inline promql/parse_test.go]: How about testing both the old and new versions here? Either just duplicate the expected result, or check the parsing of the new style, then check that the result of the old and new style is the same?

**Comment by matthiasr** [comment]
:+1:

File: promql/parse_test.go (hunk 1 above)
Comment (fabxc) [inline promql/parse_test.go]: There was no clear point to this test to begin with. Deletion wasn't due to this change.

**Comment by matthiasr** [comment]
I don't think unit tests need "a point" – just shoving a large number of examples through the parser would not hurt.

**Comment by fabxc** [comment]
Unit tests should ideally catch each code-path. If I just add a random set of tests (which also can be highly useful, i.e. fuzzing), I run at the risk of missing essential cases while thinking "I've 100 tests, this must be super-correct then".

**Pull request merged**